Doc/c-api/init.rst

```diff
@@ -1156,7 +1160,7 @@ Python-level trace functions in previous versions.
 
    Set the tracing function to *func*.  This is similar to
    :c:func:`PyEval_SetProfile`, except the tracing function does receive line-number
-   events.
+   events or any event related to C function objects being called.
```

Comment:
It is *does receive*, so I think it is wrong here. Maybe just declare it does not receive PyTrace_C_CALL/RETURN/EXCEPTION here, so don't have to write the previous added block.